gradle/jacoco.gradle

```diff
@@ -1,45 +1,36 @@
 apply plugin: 'jacoco'
 
 jacoco {
-  toolVersion = pluginVersion.jacoco
+  toolVersion "0.7.4.201502262128"
```

Comment:
Hardcoded the version here for now since sonarqube seems to have a hard time parsing the latest versions `.ec` file.